src/styles/blocks/process.scss

```diff
@@ -0,0 +1,160 @@
+.process {
+  &__description {
+    @extend %openSansSmallContent;
+    color: $gray-color;
+  }
+
+  &__cards {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+
+    @include onTablet {
+      gap: 30px;
+    }
+
+    @include onSmallDesktop {
+      margin-top: 140px;
+      gap: 20px;
+      grid-template-columns: repeat(auto-fill, 214px);
+    }
+
+    @include theDesign {
+      gap: 30px;
+      grid-template-columns: repeat(auto-fill, 264px);
+    }
+    &-bottom {
+      margin-bottom: 24px;
+
+      @include onTablet {
+        margin-bottom: 0;
+      }
+    }
+
+    &-top {
+      @include onTablet {
+        transform: translateY(-132px);
+      }
+
+      @include onSmallDesktop {
+        transform: translateY(-140px);
+      }
+
+      @include onDesktop {
+        transform: translateY(-132px);
+      }
+    }
+  }
+
+  &__card {
+    background-color: $white-color;
+    box-shadow: 0 9px 18px rgba(37, 41, 49, 0.03);
+    border-radius: 16px;
+    padding: 32px;
+    transition: all $transition-duration;
+
+    &:hover {
+      transform: translateY(-50%);
+      box-shadow: 0 0 20px $dark-blue-color;
+    }
+  }
+
+  &__num {
+    @extend %poppinsIconBlue;
+    width: 46px;
+    height: 24px;
+    border-radius: 23px;
+    background-color: $light-color;
+    text-align: center;
+    text-transform: uppercase;
+    color: $blue-color;
+
+    &--blue {
+      color: $light-blue-color;
+      background-color: $blue-gray-color;
+    }
+
+    &--yellow {
+      color: $yellow-color;
+      background-color: $dark-yellow-color;
+    }
+
+    &--green {
+      color: $green-color;
+      background-color: $dark-green-color;
+    }
+
+    &--pink {
+      color: $pink-color;
+      background-color: $dark-pink-color;
+    }
+  }
+
+  &__text {
+    @extend %openSansContent13;
+    color: $gray-color;
+    padding: 24px 0 32px;
+  }
+
+  &__learn-more {
+    @extend %poppinsButtonText;
+    line-height: 40px;
+    display: block;
+    background-image: url(../images/services/Arrow.svg);
+    background-size: 40px;
+    background-repeat: no-repeat;
+    text-decoration: none;
+    color: $gray-blue-color;
+
+    &::after {
+      content: "Learn more";
+      position: relative;
+      left: 56px;
+    }
+
+    &:hover {
+      transform: scale(1.1) translateY(-50%);
+      transition: transform 0.3s ease;
```

Comment:
Transition should be added to origin selector not :hover, so it is smooth. But to mee those process cards have too much animation and it is jumping a lot, please make it smoother and more gentle